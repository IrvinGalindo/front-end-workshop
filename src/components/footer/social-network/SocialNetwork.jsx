import React from 'react';

import PropTypes from 'prop-types';

const SocialNetwork = ({ site }) => (
  <li className="yta_Footer-socialNetwork-item">
    <a href="#">
      <span data-icon={`ei-sc-${site}`} data-size="m"></span>
      <span className="yta_u-onlyRead">{site}</span>
    </a>
  </li>
);

SocialNetwork.propTypes = {
  site: PropTypes.string.isRequired,
}

export default SocialNetwork;
