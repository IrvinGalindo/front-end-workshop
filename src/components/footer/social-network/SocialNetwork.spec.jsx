import React from 'react';
import { shallow } from 'enzyme';

import SocialNetwork from './SocialNetwork';

describe('SocialNetwork', () => {
  const socialNetwork = shallow(<SocialNetwork site="facebook" />);

  it('should render', () => {
    expect(socialNetwork).toMatchSnapshot();
  });
});
