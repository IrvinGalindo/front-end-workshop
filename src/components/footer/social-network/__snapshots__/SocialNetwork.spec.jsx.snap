// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SocialNetwork should render 1`] = `
<li
  className="yta_Footer-socialNetwork-item"
>
  <a
    href="#"
  >
    <span
      data-icon="ei-sc-facebook"
      data-size="m"
    />
    <span
      className="yta_u-onlyRead"
    >
      facebook
    </span>
  </a>
</li>
`;
